### PR TITLE
Updated stat var hierarchy to accept multiple root node filters.

### PR DESCRIPTION
**Pull request by dwnoble**
* Updated stat var hierarchy to accept multiple root node filters.
* Updated UN staging environment to filter hierarchy by two root nodes

Screenshots of various dc environments

## Base DC
base-dc

## Base Custom DC
base-custom-dc

## UNSDG Staging
unsdg-staging

## UNSDG Staging Map Tool
unsdg-staging-map

## UNSDG 
unsdg

## World Bank
worldbank

## World bank (known issue-counts are off when a place is selected)
worldbank-timeline
worldbank-timeline-ghana

Counts are off in the world bank hierarchy. This looks similar to an existing issue with https://worldbank.datacommons.org - (counts arent updating with place selection). Can address in a future PR.

### CODE REVIEW

File: server/app_env/_base.py [hunk 1]
```diff
@@ -74,3 +74,5 @@ class Config:
   MAP_TOOL_FOOTER = ""
   # The default property to use for getting geojsons
   GEO_JSON_PROP = "geoJsonCoordinates"
+  # Typedef in static/js/tools/stat_var/stat_var_hierarchy_config.ts
+  STAT_VAR_HIERARCHY_CONFIG = {"nodes": [{"dcid": "dc/g/Root"}]}
```
Comment (shifucun) [inline server/app_env/_base.py:78]: might add more comments on what fields exist and how to update.

File: server/templates/custom_dc/unsdg_staging/base.html [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+  {% if GA_ACCOUNT %}
+  <script async src="https://www.googletagmanager.com/gtag/js?id={{ GA_ACCOUNT }}"></script>
+  <script>
+    window.dataLayer = window.dataLayer || [];
+    function gtag() { dataLayer.push(arguments); }
+    gtag('js', new Date());
+    gtag('config', '{{ GA_ACCOUNT }}', { client_storage: 'none', anonymize_ip: true });
+  </script>
+  {% endif %}
+  <title>{{ title }} - UN SDG Data Commons</title>
```
Comment (shifucun) [inline server/templates/custom_dc/unsdg_staging/base.html:47]: Add "Staging"?

File: server/templates/custom_dc/unsdg_staging/base.html [hunk 3]
```diff
@@ -0,0 +1,125 @@
+{#-
```
Comment (shifucun) [inline server/templates/custom_dc/unsdg_staging/base.html:1]: how much this file can share with the unsdg/base.html? Might be helpful to make one inherit another so changes can happen just in one place.

File: static/js/stat_var_hierarchy/stat_var_hierarchy.tsx [hunk 4]
```diff
@@ -250,85 +253,85 @@ export class StatVarHierarchy extends React.Component<
...
-    if (globalThis.dataSourceDcid) {
-      allPromises.push(
+    const variableGroupInfoPromises: Promise<StatVarGroupNodeType>[] =
+      STAT_VAR_HIERARCHY_CONFIG.nodes.map((statVarHierarchyConfigNode) =>
         axios
           .post("/api/variable-group/info", {
-            dcid: ROOT_SVG,
-            entities: [globalThis.dataSourceDcid],
+            dcid: statVarHierarchyConfigNode.dcid,
+            entities: [
```
Comment (shifucun) [inline static/js/stat_var_hierarchy/stat_var_hierarchy.tsx:264]: maybe define `entities` with conditional statement to make it more explicit

File: server/app_env/_base.py (hunk 1 above)
Comment (dwnoble) [reply server/app_env/_base.py:78]: Updated!

File: server/templates/custom_dc/unsdg_staging/base.html (hunk 2 above)
Comment (dwnoble) [reply server/templates/custom_dc/unsdg_staging/base.html]: Ended up removing these templates altogether

File: static/js/stat_var_hierarchy/stat_var_hierarchy.tsx (hunk 4 above)
Comment (dwnoble) [reply static/js/stat_var_hierarchy/stat_var_hierarchy.tsx:264]: Updated

**Review by shifucun — APPROVED** [review]

**Pull request merged**